coolwsd-systemplate-setup

```diff
@@ -140,6 +140,14 @@ fi
 find $LOCALBASE/share -name '*.pcf' | xargs rm -f
 find $LOCALBASE/share -name '*.pcf.gz' | xargs rm -f
 
+# Same for /usr/local/share/fonts
+
+mkdir -p $LOCALBASE/local/share || exit 1
+${CP} -r -p -L /${LOCALBASE}/local/share/fonts $LOCALBASE/local/share
```

Comment:
There may be no /usr/local/share/fonts on the system, would it make sense to put this inside an `if [ -d /usr/local/share/fonts ]; then ... fi` block? Mostly to avoid "to be ignored" error messages during the build.

Reply:
Mike - this is lingering for age s- can you add the if and re-submit ? =)